src/marqo/core/models/update_documents.py

```diff
@@ -0,0 +1,29 @@
+from typing import Dict, Sequence, Any
+
+from pydantic import validator
+
+from marqo.api.exceptions import BadRequestError
+from marqo.base_model import ImmutableStrictBaseModel
+from marqo.tensor_search.enums import EnvVars
+from marqo.tensor_search.utils import read_env_vars_and_defaults_ints
+
+
+class UpdateDocumentsParams(ImmutableStrictBaseModel):
```

Comment:
I wonder if something like `UpdateDocumentsRequest` is more descriptive? Also module name should usually match class name